src/vos/storage_estimator/SConscript

```diff
@@ -1,37 +1,36 @@
-"""Install DAOS Storage Estimador"""
+"""Install DAOS Storage Estimator"""
 import sys
 
+
 def install_storage_estimator_module(env, lib_prefix):
-    """Install Storage Estimador module for the specified python version"""
+    """Install Storage Estimator module for the specified python version"""
 
     if GetOption("help"):
         return
 
     denv = env.Clone()
-    version = "{}.{}".format(
-        sys.version_info.major, sys.version_info.minor)
-    if sys.version_info.major == 3:
-        install_path = lib_prefix + "/python"
-        install_path += version + "/site-packages/storage_estimator"
-
-        denv.Install(install_path, "common/vos_structures.py")
-        denv.Install(install_path, "common/__init__.py")
-        denv.Install(install_path, "common/dfs_sb.py")
-        denv.Install(install_path, "common/vos_size.py")
-        denv.Install(install_path, "common/explorer.py")
-        denv.Install(install_path, "common/parse_csv.py")
-        denv.Install(install_path, "common/util.py")
-    else:
-        raise Exception("Unsupported python version %s" % version)
+    version = f'{sys.version_info.major}.{sys.version_info.minor}'
+    install_path = lib_prefix + "/python"
+    install_path += version + "/site-packages/storage_estimator"
+
+    denv.Install(install_path, "common/vos_structures.py")
+    denv.Install(install_path, "common/__init__.py")
+    denv.Install(install_path, "common/dfs_sb.py")
+    denv.Install(install_path, "common/vos_size.py")
+    denv.Install(install_path, "common/explorer.py")
+    denv.Install(install_path, "common/parse_csv.py")
+    denv.Install(install_path, "common/util.py")
```

Comment:
Certainly not introduced by this PR, but this is a good place to use `os.path.join` !

Reply:
True.  Looking at the storage_estimator code in general I'd rather touch is as little as possible. In fact I think I blacklisted it from all checks with this PR.